pyproject.toml

```diff
@@ -196,6 +196,8 @@ extend-select = [
     "YTT",           # flake8-2020
     "TID252",        # relative-imports
     "S101",          # to identify use of assert statement
+    "PT027",         # remove unittest style assertion
+    "PT009",         # Use pytest.raises instead of unittest-style
```

Comment:
Can you instead try turning on the "PT" rule commented above and see what happens?

Reply:
There are a lot of style failures. Should I try fixing those and keep the `PT` rule?

Reply:
If there are too many, then a separate PR for this would be better.

Reply:
A separate PR sounds good then. Could you create one after this, @prady0t?